modules/webform/form_builder_webform.components.inc

```diff
@@ -1051,6 +1051,7 @@ function _form_builder_webform_build_edit_form($component_type, $element, $prope
   // checked.
   $node = !isset($nid) ? (object) array('nid' => NULL, 'webform' => webform_node_defaults()) : node_load($nid);
   $form = webform_component_edit_form($empty_form, $empty_form_state, $node, $component);
+  drupal_alter(['form', 'form_webform_component_edit_form'], $form, $empty_form_state);
```

Comment:
This should also pass the form_build_id otherwise `hook_form_alter()` implementations might get confused.